Dockerfile

```diff
@@ -7,15 +6,19 @@ WORKDIR $BUILD_HOME
 USER root
 RUN ./gradlew -x test build application:shadowJar
 
+RUN ls $BUILD_HOME/application/build/libs/
```

Comment:
Remove this :)

Reply:
Thanks for noticing this.
Happy new year xD